.github/workflows/ci.yml

```diff
@@ -356,7 +356,7 @@ jobs:
               --no-update-lock-file \
               --print-out-paths \
               '.#packages.${{ matrix.system }}.flox-cli-tests'
-          nix copy --to "$FLOX_SUBSTITUTER" ./result
+          nix copy --extra-experimental-features 'nix-command' --to "$CONFIGURE_NIX_SUBSTITUTER" ./result
```

Comment:
question(non-blocking): Would it make sense for the `configure-nix-action` to set these experimental features by default?